### PR TITLE
replaced pictures for fan box

### PR DESCRIPTION
**Pull request by Venkat4real**
Continuation of #125

### CODE REVIEW

File: content/en/v1.1/fan-box.adoc [hunk 1]
```diff
@@ -43,77 +43,77 @@ These parts should be printed at 0.2 or 0.15 spacing.  They should be oriented s
...
-| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000159%20Stealth%20Filter%20Frame%20Sanyo.stl
+| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000159%20Stealth%20Filter%20Frame%20Sanyo.stl[STL]
 
 | Weather Guard
 | P-TET-000143
 | 1
 | PLA/PETG
 | No
-| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000159%20Stealth%20Filter%20Frame%20Sanyo.stl
+| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000159%20Stealth%20Filter%20Frame%20Sanyo.stl[STL]
```
Comment (mmroden) [inline content/en/v1.1/fan-box.adoc:67]: This is not the weather gard, but it's the filter frame. https://github.com/tetrabiodistributed/papr-print-files/blob/main/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000143%20Weather%20Guard.stl (but the permalink) is correct

File: content/en/v1.1/fan-box.adoc [hunk 2]
```diff
@@ -43,77 +43,77 @@ These parts should be printed at 0.2 or 0.15 spacing.  They should be oriented s
 | 1
 | PLA/PETG
 | No (Can have some in the nut pocket)
-| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000140%20Base%20Sanyo.st
+| https://github.com/tetrabiodistributed/papr-print-files/blob/f77c5bc3c5f0d8fadfdf2757c59a09a8cd5e584b/STLs/fan_box/96mm%20Stealth%20Parts/P-TET-000140%20Base%20Sanyo.st[STL]
```
Comment (mmroden) [inline content/en/v1.1/fan-box.adoc:46]: Link should end with .stl

File: content/en/v1.1/fan-box.adoc [hunk 3]
```diff
@@ -167,22 +167,22 @@ These are components that need to be purchased.  Prices fluctuate, and so are no
 | Count | Description | URL | Number in Package 
 | 1 
 | Sanyo Denki 9BMB12F201
-| https://octopart.com/9bmb12f201-sanyo+denki-24320259
+| https://octopart.com/9bmb12f201-sanyo+denki-24320259[Link]
```
Comment (mmroden) [inline content/en/v1.1/fan-box.adoc:170]: Maybe link to the main electronics BOM?  I don't think this is the right fan, but maybe it is.  I thought it ended with 202 not 201

**Review by mmroden — CHANGES_REQUESTED** [review]
Minor changes

File: content/en/v1.1/fan-box.adoc (hunk 2 above)
Comment (Burhan-Q) [reply content/en/v1.1/fan-box.adoc:46]: @Venkat4real this link needs to fixing still

File: content/en/v1.1/fan-box.adoc [hunk 4]
```diff
@@ -197,12 +197,12 @@ These are components that need to be purchased.  Prices fluctuate, and so are no
...
 | 1
 | BNC 20 gauge wire stays
-| https://www.jameco.com/z/205-523-Steren-Electronics-Cable-Assembly-RG58-U-36-Bnc-To-Bnc-50-Ohm_11404.html
+| https://www.jameco.com/z/205-523-Steren-Electronics-Cable-Assembly-RG58-U-36-Bnc-To-Bnc-50-Ohm_11404.html[Link]
 | 1
 
 | 1
 |Stealth Filters
-|https://www.amazon.com/STEALTH-Respirator-Filters-Woodworking-Replacement/dp/B09H52LGGW[Stealth Filters]
+|https://www.amazon.com/STEALTH-Respirator-Filters-Woodworking-Replacement/dp/B09H52LGGW[link]
```
Comment (Burhan-Q) [inline content/en/v1.1/fan-box.adoc:205]: Maybe make `[Link]` (capitalized "L") for consistency

**Comment by Burhan-Q** [comment]
@Venkat4real looks like there's a few fixes needed, but once addressed this can be merged.

**Review by Burhan-Q — CHANGES_REQUESTED** [review]

**Comment by Venkat4real** [comment]
Closing this PR because the changes are made in the last PR.

**Pull request closed without merging**